app/portainer/components/DetailsTable/DetailsTableKeyValueRow.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactNode } from 'react';
+
+interface Props {
+  children?: ReactNode;
+  keyProp: string;
+}
+
+export function DetailsTableKeyValueRow({ keyProp, children }: Props) {
```

Comment:
this component will only be imported in details table components. we can call it `Row` or `DetailsRow`